packages/flutter_nekoton_bridge/example/integration_test/core/keystore_test.dart

```diff
@@ -0,0 +1,414 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_nekoton_bridge/flutter_nekoton_bridge.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+class MockedStorageMethods {
+  final data = <String, String>{};
+
+  Future<String?> get(String key) async {
+    return data[key];
+  }
+
+  Future<void> set({
+    required String key,
+    required String value,
+  }) async {
+    data[key] = value;
+  }
+
+  void setUnchecked({
+    required String key,
+    required String value,
+  }) {
+    data[key] = value;
+  }
+
+  Future<void> remove(String key) async {
+    data.remove(key);
+  }
+
+  void removeUnchecked(String key) {
+    data.remove(key);
+  }
+}
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  late MockedStorageMethods storageMethods;
+  const signers = [KeySigner.encrypted(), KeySigner.derived()];
+
+  const phraseLabs =
+      'company conduct cave custom ill fox loop type holiday veteran clog oxygen';
+  const phraseLegacy =
+      'country glue knife buzz bus armor cement offer guide corn buddy update bird alcohol either neglect demand uncover table lock ketchup dinner ramp cream';
+  const password = 'password';
+
+  const inputLabsData = DerivedKeyCreateInputImport(
+    keyName: 'KeyNameLabs',
+    phrase: phraseLabs,
+    password: Password.explicit(
+      PasswordExplicit(
+        password: password,
+        cacheBehavior: PasswordCacheBehavior.nop(),
+      ),
+    ),
+  );
+  const addKeyInputLabs = DerivedKeyCreateInput.import(inputLabsData);
+
+  final addKeyInputLegacy = EncryptedKeyCreateInput(
+    name: 'KeyNameLegacy',
+    mnemonicType: const MnemonicType.legacy(),
+    phrase: phraseLegacy,
+    password: const Password.explicit(
+      PasswordExplicit(
+        password: password,
+        cacheBehavior: PasswordCacheBehavior.nop(),
+      ),
+    ),
+  );
+
+  setUp(() {
+    storageMethods = MockedStorageMethods();
+    // This setup thing SHOULD NOT be removed or altered because it used in integration tests
+    setupLogger(
+      level: LogLevel.Trace,
+      mobileLogger: false,
+      logHandler: (logEntry) => debugPrint(
+        'FromLib: ${logEntry.level} ${logEntry.tag} ${logEntry.msg} (lib_time=${logEntry.timeMillis})',
+      ),
+    );
+  });
+
+  group('KeyStore test', () {
+    testWidgets('Create KeyStore', (WidgetTester tester) async {
+      runApp(Container());
+
+      await tester.pumpAndSettle();
+      await initRustToDartCaller();
```

Comment:
i'm not sure, but maybe we can use setUpAll() for runApp(), first pumpAndSettle() and initRustToDartCaller()?

Reply:
Good idea

Reply:
Hmm, I tried to do this, but we have some troubles: 
1) `tester. pumpAndSettle()` can't be moved to `setUp` function because of missed `tester`.
2) Moving `storage` to `setUp` function causes failing tests, idk why.

Reply:
`runApp` and `initRustToDartCaller` had been moved